browser/main/lib/modal.js

```diff
@@ -15,9 +15,13 @@ class ModalBase extends React.Component {
 
   close () {
     if (modalBase != null) modalBase.setState({component: null, componentProps: null, isHidden: true})
-    // Toggle overflow style on NoteList
-    let list = document.querySelector('.NoteList__list___browser-main-NoteList-')
-    list.style.overflow = 'auto'
+
+    // Toggle display for scrollbars
+    let list = document.querySelectorAll('.NoteList__list___browser-main-NoteList-, .SnippetNoteDetail__description___browser-main-Detail- > textarea')
```

Comment:
Hi @gabber12 !  I am a Boostnote maintainer . We want to improve the code which modal know other components. So, it is hard to merge this PR, now. Also, We'll discuss the matter. However, it would be very appreciated if you would improve it.